include/seqan3/search/configuration/detail.hpp

```diff
@@ -27,13 +27,13 @@ namespace seqan3::detail
  * \details
  *
  * The seqan3::detail::search_config_id used to identify a specific search configuration element independent of
- * it's concrete type and position within the seqan3::search_cfg::search_configuration object.
+ * it's concrete type and position within the seqan3::search_cfg object.
```

Comment:
Right now, the link points to the namespace, which I fell a little misleading. Instead I would only name the search configuration and link it to the module page of the search configuration. This landing page should then be used to explain more in depth the principles and functionality of the search.

Reply:
I am sorry, I don't get what I should change here.

Reply:
```suggestion
 * its concrete type and position within the \ref seqan3::search_cfg::search_configuration "search configuration object".
```